### PR TITLE
feat: tanka-util

### PR DESCRIPTION
**Pull request by Duologic**
- [x] refactor helm-util into tanka-util
- [x] add Kustomize support
- [x] deprecate helm-util, but still provide it
- [x] use tanka-util in cert-manager

### CODE REVIEW

File: tanka-util/README.md [hunk 1]
```diff
@@ -2,28 +2,30 @@
 permalink: /
 ---
 
-# package helm-util
+# package tanka-util
 
 ```jsonnet
-local helm-util = import "github.com/grafana/jsonnet-libs/helm-util/helm.libsonnet"
+local tanka-util = import "github.com/grafana/jsonnet-libs/tanka-util/main.libsonnet"
```
Comment (malcolmholmes) [inline tanka-util/README.md:8]: `local tanka-util` requires an underscore.

File: tanka-util/README.md.tmpl [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
 ### Usage
 
-> **Warning:** [Functionality required](#internals) by this library is still experimental and may break.
+> **Warning:** [Functionality required](#internals) by this library is still
+> experimental and may break.
 
-The [`helm.template`](#fn-helmtemplate) function to converts a Helm Chart into Jsonnet objects,
-to be consumed by tools like [Grafana Tanka](https://tanka.dev).
+The [`helm.template`](#fn-helmtemplate) function converts a Helm Chart into
+Jsonnet object to be consumed by tools like Tanka.
```
Comment (malcolmholmes) [inline tanka-util/README.md.tmpl:11]: Into *a* Jsonnet object

File: tanka-util/README.md [hunk 3]
```diff
@@ -2,28 +2,30 @@
...
 ```
 
-Package `helm-util` allows the user to consume Helm Charts as plain Jsonnet
-resources. This package implements [Helm support](https://tanka.dev/helm) for
-Grafana Tanka.
+Package `tanka-util` provides jsonnet tooling that works well with [Grafana
+Tanka](https://tanka.dev) features. This package implements [Helm
```
Comment (malcolmholmes) [inline tanka-util/README.md:12]: For example, this package implements...

**Review by malcolmholmes — APPROVED** [review]
Looks great, with a few nits

**Comment by Duologic** [comment]
This is a draft, don't review that. 🙃

File: tanka-util/README.md (hunk 1 above)
Comment (Duologic) [reply tanka-util/README.md:8]: This is autogenerated by docsonnet :grimacing:

File: tanka-util/main.libsonnet [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    url='github.com/grafana/jsonnet-libs/tanka-util/main.libsonnet',
+    help=(importstr 'README.md.tmpl') % (importstr '_example.jsonnet'),
+  ),
+
+  '#util':: d.obj(
+    |||
+      `util` provides common utils to modify Kubernetes objects.
+    |||
+  ),
+  util: (import 'util.libsonnet'),
```
Comment (sh0rez) [inline tanka-util/main.libsonnet:16]: Can we name this `k8s` instead? `util` is a term that carris few useful information

**Review by sh0rez — APPROVED** [review]

File: tanka-util/README.md (hunk 1 above)
Comment (malcolmholmes) [reply tanka-util/README.md]: But it is invalid jsonnet, so needs fixing :-)

**Review by malcolmholmes — APPROVED** [review]

**Pull request merged**